core/pkg/runtime/runtime.go

```diff
@@ -37,8 +37,7 @@ func (r *Runtime) Start() error {
 	if r.Evaluator == nil {
 		return errors.New("no evaluator set")
 	}
-	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
-	defer cancel()
```

Comment:
please, keep the cancel function since some function down the line could listen to the `done()` event